.codesandbox/ci.json

```diff
@@ -3,6 +3,7 @@
   "installCommand": "install:codesandbox",
   "node": "14",
   "packages": [
+    "packages/x-codemod",
```

Comment:
Added it here initially just for testing. I don't think we need to keep releasing it on every pipeline. Would you agree?